docs/modules/storage.md

```diff
@@ -8,6 +8,7 @@ _Note: This module is EXPERIMENTAL. To enable this module, you must run Open Ser
 
 | Plan Name | Description |
 |-----------|-------------|
+| `general-purpose-v2-storage-account` | This is the second generation storage account and is recommended for most scenarios. This plan provisions a general purpose account only. Create your own containers, files, and tables within this account. |
 | `general-purpose-storage-account` | Provisions a general purpose account only. Create your own containers, files, and tables within this account. |
 | `blob-storage-account` | Provisions a blob storage account only. Create your own blob containers (only) within this account. |
 | `blob-container` | Provisions a blog storage account and a blob container within. |
```

Comment:
This plan is for `general-purpose-storage-account`. Should we remove it or add a similar one for `general-purpose-v2-storage-account`?

Reply:
You mean `blob-container`? This plan is for `blob-storage-account`.

Reply:
Oh, I got it. Thanks for correcting me.